examples/typescript/backends/remote/package.json

```diff
@@ -1,10 +1,11 @@
 {
-  "name": "@deactivated-examples/typescript-backends-remote",
+  "//": "This example test is disabled via the 'private' attribute since it needs credentials (I assume at least)",
```

Comment:
Actually using it would need credentials, but just synthing doesn't need anything.